packages/houdini/package.json

```
@@ -61,7 +60,7 @@
         "vite-plugin-watch-and-run": "^1.7.0"
     },
     "peerDependencies": {
-        "vite": "^5.3.3"
+        "vite": "^6.0.3"
```

i think this would be a breaking change. if you want to add vite6, that can be done in a minor, but removing vite5 would be breaking. (a user stuck on vite5 cannot update houdini anymore)

---

Hmm that's fair. So something like this would be a better solution then?
```suggestion
        "vite": "^5.3.3 || ^6.0.3"
```